pkgs/development/libraries/tcl-fcgi/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, fetchFromGitHub, tcl, tclx }:
+
+tcl.mkTclDerivation rec {
+  pname = "tcl-fcgi";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "mpcjanssen";
+    repo = "tcl-fcgi";
+    rev = "62452db";
```

```suggestion
    rev = "62452dbf3177ba9458fbb42457834ca77bdf5a82";
```
we must use long hashes